src/editor/components/preview/ReactMaterialPreview.tsx

```diff
@@ -11,20 +11,21 @@ import {
   materialRenderers,
 } from '@jsonforms/material-renderers';
 import { JsonForms } from '@jsonforms/react';
-import React from 'react';
+import React, { useMemo } from 'react';
 
 import { useExportSchema, useExportUiSchema } from '../../../core/util/hooks';
+import { generateDataForPreview } from './data';
 import { previewOptions } from './options';
 
 export const ReactMaterialPreview: React.FC = () => {
   const schema = useExportSchema();
   const uischema = useExportUiSchema();
-
+  const previewData = useMemo(() => generateDataForPreview(schema), [schema]);
   const ajv = createAjv(previewOptions);
   return (
     <JsonForms
       ajv={ajv}
-      data={{}}
```

Comment:
the old code for data is enough

Reply:
Validation/defaults don't work in react either without the initialized object.